src/pybind/mgr/orchestrator/module.py

```diff
@@ -380,6 +380,13 @@ def _get_hosts(self, format: Format = Format.plain) -> HandleCommandResult:
             output = table.get_string()
         return HandleCommandResult(stdout=output)
 
+    @_cli_read_command('orch host gather facts')
+    def _get_hosts_gather_facts(self, hostname) -> HandleCommandResult:
+        """List hosts metadata"""
+        completion = self.get_gather_facts(hostname)
```

Comment:
this will probably not going to work with mypy. You'd need to declare the method in the `Orchestrator` interface class first.

Reply:
Done. Thanks